lib/feform.js

```diff
@@ -51,7 +51,10 @@ var findInputs = function findInputs(children, valueChange) {
     _react2.default.Children.forEach(children, function (child) {
         // have to assign a key to avoid warning about dynamic children
         // (https://facebook.github.io/react/docs/multiple-components.html#dynamic-children)
-        var childKey = _lodash2.default.isNil(child.key) ? 'feinput-child' + context.childIndex++ : child.key;
+        if (_lodash2.default.isNil(child)) {
```

Comment:
Sorry, should've told you not to modify this file - it gets auto-generated before publishing.